widgetssdk/src/main/java/com/glia/widgets/secureconversations/SecureConversations.kt

```diff
@@ -42,16 +61,34 @@ class SecureConversationsImpl(
     private val secureConversations: com.glia.androidsdk.secureconversations.SecureConversations
 ) : SecureConversations {
 
-    override fun getUnreadMessageCount(callback: RequestCallback<Int?>) {
-        secureConversations.getUnreadMessageCount(callback)
+    internal val subscribedCallbacks: MutableMap<Int, RequestCallback<Int>> = mutableMapOf()
```

Comment:
I'm not complaining, but why not:
```suggestion
    internal val subscribedCallbacks: MutableMap<OnSuccess<Int>, RequestCallback<Int>> = mutableMapOf()
```

Reply:
The reasons for going with an Int hash code were:
1. Efficiency: Using hash codes as keys is generally more memory-efficient and faster for lookups, as hash codes are simple integers.
2. Simplicity in Storage: Storing an Int instead of the full object reduces the memory footprint of the map.

However, the expected number of `RequestCallback`  should be small and performance should not be a critical concern, I agree. So, I can change it to `MutableMap<OnSuccess<Int>, RequestCallback<Int>>` however, I don't see strong reasons for keeping the whole object as a key. 🤔

Reply:
Yes, it can be memory-efficient. Also, we don't store a link to the object. However we store `RequestCallback`s.
This is a little more convenient to use, because it is difficult to confuse an object.

But, let it be `Int`.